src/features/discord/messages/classes/commands/flags/discord-command-boolean-flag.ts

```diff
@@ -14,4 +18,64 @@ export class DiscordCommandBooleanFlag<
   public constructor(discordCommandFlag: Readonly<IDiscordCommandFlag<T>>) {
     super(discordCommandFlag);
   }
+
+  /**
+   * @description
+   * Check if the flag value is a boolean
+   *
+   * @example
+   * isValid('enabled')       => false
+   * isValid('enabled=')      => false
+   * isValid('enabled=dummy') => false
+   * isValid('enabled=true')  => true
+   * isValid('enabled=false') => true
+   *
+   * @param {Readonly<string>} messageFlag A flag as a message
+   *
+   * @return {boolean} true if the flag value is valid
+   */
+  public isValid(messageFlag: Readonly<string>): boolean {
+    const splittedFlag: string[] = _.split(messageFlag, `=`);
+    const splittedFlagSize: number = _.size(splittedFlag);
+
+    if (_.lt(splittedFlagSize, 2)) {
```

Comment:
2 must be a constant in a variable, right?

Reply:
@vvalembois why is that?

Reply:
Magic numbers, will do in the next PR.